fastapi/concurrency.py

```diff
@@ -1,6 +1,6 @@
 from typing import Any, Callable
 
-from starlette.concurrency import iterate_in_threadpool, run_in_threadpool  # noqa
```

Comment:
Actually, I was planning on importing everything from Starlette, and making it available from FastAPI, so that users (and even myself) don't have to guess what to import from FastAPI and what from Starlette.

And as I was creating that "context manager in threadpool" that was not provided by Starlette, I thought of including `iterate_in_threadpool` right away...

Anyway, that's the reason. But I'll finish that in a PR later. :sweat_smile: